src/resolvers/electrum_blocking.rs

```diff
@@ -78,10 +78,19 @@ impl RgbResolver for Client {
 
         // Now we get and parse transaction information to get the number of
         // confirmations
-        let tx_details = check!(self.raw_call("blockchain.transaction.get", vec![
+        let tx_details = match self.raw_call("blockchain.transaction.get", vec![
```

Comment:
More rust way is to use `.or_else(|err| if err.to_string().containes("...") { return Ok(witness_anchor) } else { err })` instead of match

Reply:
@dr-orlovsky This will not work because `or_else` expects the same return type as the original `Result`.